patches/server/0010-Adventure.patch

```diff
@@ -846,17 +693,13 @@ index 0000000000000000000000000000000000000000..57081e84d50f7eca7ea11540b550da82
 +         */
 +        this.message = message;
 +        this.async = async;
-+        if (this.message.requireResult().modernized()) {
-+            this.craftbukkit$originalMessage = this.message.requireResult().message().legacyMessage();
-+        } else {
-+            this.craftbukkit$originalMessage = message.signedContent();
-+        }
++        this.craftbukkit$originalMessage = message.signedContent();
```

Comment:
It might not be appropriate to have this anymore? And just pass the legacy serialized result from ChatDecorator (or the plain string if there isn't one).

Reply:
Leaving that out of this pull request as discussed.